contracts/debt/IncurDebtV2.sol

```diff
@@ -11,18 +11,18 @@ import "./interfaces/IStrategy.sol";
 import "../libraries/SafeERC20.sol";
 import "../types/OlympusAccessControlledV2.sol";
 
-error IncurDebtV1_NotBorrower(address _borrower);
-error IncurDebtV1_InvaildNumber(uint256 _amount);
-error IncurDebtV1_WrongTokenAddress(address _token);
-error IncurDebtV1_AlreadyBorrower(address _borrower);
-error IncurDebtV1_AboveGlobalDebtLimit(uint256 _limit);
-error IncurDebtV1_AboveBorrowersDebtLimit(uint256 _limit);
-error IncurDebtV1_LimitBelowOutstandingDebt(uint256 _limit);
-error IncurDebtV1_AmountAboveBorrowerBalance(uint256 _amount);
-error IncurDebtV1_AmountMoreThanBorrowersLimit(uint256 _borrower);
-error IncurDebtV1_OHMAmountMoreThanAvailableLoan(uint256 _amount);
-error IncurDebtV1_BorrowerHasNoOutstandingDebt(address _borrower);
-error IncurDebtV1_BorrowerStillHasOutstandingDebt(address _borrower);
+error IncurDebtV2_NotBorrower(address _borrower);
```

Comment:
Maybe just remove the V2 denotation completely.

Reply:
yea i want to make v1 and v2 1 contract we'll discuss in meeting